modules/tensor_mechanics/test/tests/shell/static/abaqus_plate_bending.i

```diff
@@ -0,0 +1,208 @@
+# Shell element verification test from Abaqus verification manual 1.3.13
+
+# A 40 m x 20 m x 1 m plate that has E = 1000 Pa and Poisson's ratio = 0.3
+# is subjected to the following boundary/loading conditions. A single shell
+# element is used to model the plate.
+
+# disp_z = 0 at vertices A (0, 0), B (40, 0) and D (20, 0).
+# disp_x and disp_y are zero at all four vertices.
+
+# F_z = -2.0 N at vertex C (40, 20).
+# M_x = 20.0 Nm at vertices A and B (bottom boundary)
+# M_x = -20.0 Nm at vertices C and D (top boundary)
+# M_y = 10.0 Nm at vertices B and C (right boundary)
+# M_y = -10.0 Nm at vertices A and D (left boundary)
+
+# The disp_z at vertex C is -12.54 m using S4 elements in Abaqus.
+# The solution obtained using Moose is -12.519 m with a relative error
+# of 0.16%.
+
+[Mesh]
+  [./gmg]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 1
+    ny = 1
+    xmin = 0.0
+    xmax = 40.0
+    ymin = 0.0
+    ymax = 20.0
+  [../]
+
+  [./c_node]
+    type = ExtraNodesetGenerator
+    input = gmg
+    new_boundary = 100
+    coord = '40.0 20.0'
+  [../]
+[]
+
+[Variables]
+  [./disp_x]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./disp_y]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./disp_z]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./rot_x]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+  [./rot_y]
+    order = FIRST
+    family = LAGRANGE
+  [../]
+[]
+
+[BCs]
+  [./simply_support_x]
+    type = PresetBC
+    variable = disp_x
+    boundary = 'right top bottom left'
+    value = 0.0
+  [../]
+  [./simply_support_y]
+    type = PresetBC
+    variable = disp_y
+    boundary = 'right top bottom left'
+    value = 0.0
+  [../]
+  [./simply_support_z]
+    type = PresetBC
+    variable = disp_z
+    boundary = 'bottom left'
+    value = 0.0
+  [../]
+[]
+
+[NodalKernels]
+  [./force_C]
+    type = ConstantRate
```

Comment:
Do we really use an object called ConstantRate to apply nodal forces?!  That's a super obtuse name!

Reply:
Yes, this is an interesting name :)